api/v0/vms/vms.proto

```diff
@@ -46,25 +46,58 @@ service VmService {
 
   // Stop one or more VMs.
   rpc Stop(VmServiceStopRequest) returns (VmServiceStopResponse) {}
+
+  // List all VMs
+  rpc List(VmServiceListRequest) returns (VmServiceListResponse) {}
+}
+
+message VmServiceListRequest{}
+message VmServiceListResponse{
+  repeated VirtualMachineSummary machines = 1;
+}
+
+message VirtualMachineSummary {
+  // The identifier of the VM
+  string id = 1;
+
+  // Status of the VM
+  string status = 2;
+
+  // The memory size of VM
+  uint32 mem_size_mb = 3;
+
+  // The number of vCPUs for the VM
+  uint32 vcpu_count = 4;
+
+  // The path to the VM kernel image
+  string kernel_img_path = 5;
+
+  // Path to the image mounted as the root directory of the VM
+  string root_dir_path = 6;
+
+  // Auraed server address of the VM
```

Comment:
does this need a  port or is it host:port?  ipv4 or ipv6?

Reply:
This is currently a link-local ipv6 host:port which is allocated regardless of whether there actually is an auraed set to run on the VM.

It may be best to potentially remove all "nested" auraed assumptions in this PR and just get an initial baseline of "creating VMs" before we come together and decide out how we want to handle auraed in VMs.